src/components/Table/TasksTable.stories.tsx

```diff
@@ -39,55 +41,11 @@ const tasks = [
     baseTask: {
       status: "failed",
     },
+    execution: 0,
   },
 ];
 
-const nestedTasks = [
-  ...tasks,
-  {
-    id: "some_id_4",
-    aborted: false,
-    displayName: "Some Fancy Display Task",
-    version: "234",
-    status: "success",
-    buildVariant: "Windows",
-    buildVariantDisplayName: "Windows 97",
-    blocked: false,
-    baseTask: {
-      status: "failed",
-    },
-    executionTasksFull: [
-      {
-        id: "some_id_5",
-        aborted: false,
-        displayName: "Some fancy execution task",
-        version: "234",
-        status: "success",
-        buildVariant: "Windows",
-        buildVariantDisplayName: "Windows 97",
-        blocked: false,
-        baseTask: {
-          status: "aborted",
-        },
-      },
-      {
-        id: "some_id_6",
-        aborted: false,
-        displayName: "Another execution task",
-        version: "234",
-        status: "success",
-        buildVariant: "Windows",
-        buildVariantDisplayName: "Windows 97",
-        blocked: false,
-        baseTask: {
-          status: "system-failed",
-        },
-      },
-    ],
-  },
-];
 export const BaseTaskTable = () => <TasksTable tasks={tasks} />;
-export const ExecutionTasksTable = () => <TasksTable tasks={nestedTasks} />;
```

Comment:
Why did you delete this story I think there's value in having both stories since we no longer can test collapsing and uncollapsing.